modules/FindMagnumExtras.cmake

```diff
@@ -12,7 +12,7 @@
 #
 # This command alone is useless without specifying the components:
 #
-#  (none yet)
+#  Octree - Octree BSP structure for Magnum::SceneGraph
```

Comment:
I would just say "Octree library" here and align the `- ...` at the same column as the above list